packages/calypso-e2e/src/lib/components/sidebar-component.ts

```diff
@@ -82,13 +82,9 @@ export class SidebarComponent {
 		}
 
 		// Verify the expected item or subitem is selected.
-		await Promise.all( [
-			this.page.waitForSelector( selectedMenuItem, {
-				timeout: 10000,
-				state: 'attached',
-			} ),
-			this.page.waitForSelector( '.focus-content' ),
-		] );
+		await this.page.waitForSelector( '.focus-content' );
```

Comment:
If we're waiting for the `selectedMenuItem` locator to be resolved, why do we need to wait for the `.focus-content` element as well?

Reply:
I believe it was originally meant to be a guard for the SidebarComponent animation when in mobile viewport.

Reply:
Removed in 923d20f8ca25a27b0bc5a93b11cf845f7cddd6c1.